src/Resources/translations/api_problem.en.xlf

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xliff xmlns="urn:oasis:names:tc:xliff:document:2.0" version="2.0" srcLang="zxx" trgLang="en">
```

Comment:
`srcLang` is required, not sure what to put when using translation keys... (`zxx` means 'no linguistic information', which seems odd.)